docusaurus/docs/vitest/reference.md

```diff
@@ -0,0 +1,240 @@
+- `true`: if the triplet `(branch name, commit hash, number of nodes)` was already split in a previous build use the same split, otherwise generate a new split
+
+Recommended:
+
+- `true` when your CI allows retrying single CI nodes (e.g., Buildkite, GitHub Actions) or if your CI nodes are spot instances/preemptible
+- `true` when your CI uses the same `KNAPSACK_PRO_CI_NODE_BUILD_ID` on retries (e.g., GitHub Actions, Travis, CodeShip)
+- `false` otherwise
+
+## `KNAPSACK_PRO_LOG_LEVEL`
+
+Default: `info`
+
+Available:
+
+- `error`: critical errors
+- `warn`: warnings (e.g. Fallback Mode has started)
+- `info`: Knapsack Pro API request response body
+- `verbose`
+- `debug`: Knapsack Pro API request headers and body
+- `silly`
+
+## `KNAPSACK_PRO_TEST_FILE_EXCLUDE_PATTERN`
+
+Exclude tests matching a pattern. It can be used in tandem with `KNAPSACK_PRO_TEST_FILE_PATTERN`.
+
+Default: if not specified, it falls back to [Vitest `exclude`](https://vitest.dev/config/#exclude)
+
+Available: anything that [node-glob](https://github.com/isaacs/node-glob#glob-primer) accepts
+
+Hint: you can debug in `node`
+
+```js
+var glob = require("glob");
+var MY_GLOB = "{**/__tests__/**/*.js?(x),**/?(*.)(spec|test).js?(x)}";
+glob(MY_GLOB, {}, function (err, files) {
+  console.log(files);
+});
+```
+
+Examples:
+
+```bash
+KNAPSACK_PRO_TEST_FILE_EXCLUDE_PATTERN="**/__tests__/admin/**/*.js"
+# or
+KNAPSACK_PRO_TEST_FILE_PATTERN=="{**/__tests__/**/*.js?(x),**/?(*.)(spec|test).js?(x)}" \
+KNAPSACK_PRO_TEST_FILE_EXCLUDE_PATTERN="**/__tests__/admin/**/*.js"
+```
+
+## `KNAPSACK_PRO_TEST_FILE_LIST_SOURCE_FILE`
+
+File containing the list of **relative paths** of tests to run. When `KNAPSACK_PRO_TEST_FILE_LIST_SOURCE_FILE` is set, both `KNAPSACK_PRO_TEST_FILE_PATTERN` and `KNAPSACK_PRO_TEST_FILE_EXCLUDE_PATTERN` are ignored.
+
+Default: `undefined`
+
+Example:
+
+```bash
+KNAPSACK_PRO_TEST_FILE_LIST_SOURCE_FILE=__tests__/fixtures/list.txt
+
+# ✅ list.txt
+__tests__/a.test.js
+__tests__/b.test.js
+__tests__/c.test.js
+
+# ⛔️ list.txt
+/home/user123/project/__tests__/a.test.js
+/home/user123/project/__tests__/b.test.js
+/home/user123/project/__tests__/c.test.js
+```
+
+## `KNAPSACK_PRO_TEST_FILE_PATTERN`
+
+:::caution
+
+Make sure to match individual files by adding the suffix (e.g., `.js`) so that Knapsack Pro can split by file and not by directory.
+
+:::
+
+Run tests matching a pattern. It can be used in tandem with `KNAPSACK_PRO_TEST_FILE_EXCLUDE_PATTERN`.
+
+Default: if not specified, it falls back to [Vitest `include`](https://vitest.dev/config/#include)
+
+Available: anything that [node-glob](https://github.com/isaacs/node-glob#glob-primer) accepts
+
+Hint: you can debug in `node`
+
+```js
+var glob = require("glob");
+var MY_GLOB = "{**/__tests__/**/*.js?(x),**/?(*.)(spec|test).js?(x)}";
```

Comment:
`#recommendation`

I think we should use a default pattern `**/*.{test,spec}.?(c|m)[jt]s?(x)` specific to Vitest in this example.

```suggestion
var MY_GLOB = "**/*.{test,spec}.?(c|m)[jt]s?(x)";
```